src/Bicep.Core/Parser/Parser.cs

```diff
@@ -680,6 +680,10 @@ private SyntaxBase Array()
                 if (itemOrToken is ArrayItemSyntax)
                 {
                     // items must be followed by newlines
```

Comment:
>newlines [](start = 49, length = 8)

Should probably move the comment down because it doesn't really fit here anymore.

Reply:
Same in the other one below.

---
In reply to: [512388158](https://github.com/Azure/bicep/pull/736#discussion_r512388158) [](ancestors = 512388158)